Formula/r/rsyncy.rb

```diff
@@ -0,0 +1,44 @@
+class Rsyncy < Formula
+  include Language::Python::Virtualenv
+
+  desc "Status/progress bar for rsync"
+  homepage "https://github.com/laktak/rsyncy"
+  url "https://github.com/laktak/rsyncy/archive/refs/tags/v0.2.0.tar.gz"
+  sha256 "35b8998e5ff13edd5ea833fcb8b837013424d86c3110091028d3ca71cdd46bde"
+  license "MIT"
+  head "https://github.com/laktak/rsyncy.git", branch: "master"
+
+  depends_on "pyinstaller" => :build
+  depends_on "python@3.12" => :build
+  depends_on "rsync"
+
+  uses_from_macos "zlib"
+
+  def python3
+    "python3.12"
+  end
+
+  def install
+    virtualenv_create(buildpath, python3)
+    ENV.append_path "PYTHONPATH", buildpath/Language::Python.site_packages(python3)
+    pyinstaller = Formula["pyinstaller"].opt_bin/"pyinstaller"
+    system pyinstaller, "rsyncy.py", "--onefile", "--name", "rsyncy", "--console"
+    bin.install "dist/rsyncy"
```

Comment:
This will bundle it's own python which means it goes against: https://docs.brew.sh/Acceptable-Formulae#stuff-that-requires-vendored-versions-of-homebrew-formulae

Reply:
I actually took that from an existing formula. I've changed it to require python.